repo/packages/A/arangodb3/5/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
As of packagingVersion 3.0, `command.json` is deprecated in favor of `.cli` property in resource.json 
https://github.com/mesosphere/universe#commandjson